dist/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link href="https://fonts.googleapis.com/css?family=Faster+One" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>TvShow</title>
+<script defer src="main.js"></script></head>
+<body>
+<<<<<<< HEAD
+  <div class="header">
+    <div class="logo-image">
+      <!-- <img src="/images/Film-Logo-Templates.jpg" class="logo" alt=""> -->
+      <h1 class="logo">Logo</h1>
+    </div>
+    <div class="menu">
+      <ul class="menu-item">
+        <li class="menu-link">Item1</li>
+        <li class="menu-link">Item2</li>
+        <li class="menu-link">Item3</li>
+        <li class="movie-num">Item3</li>
+      </ul>
+    </div>
+
+  </div>
+  <div class="content">
+    <!-- The Data Comes Dynamically  -->
+    <ul class="movie-list"></ul>
+  </div>
+
+  <div class="pupop-comments">
+    <!-- The Data comes Dynamically -->
+  </div>
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly fix the `margin-bottom` of your comments popup. Right now it appears to be overflowing and not appearing as intended on the entire screen.
![overflow](https://user-images.githubusercontent.com/63917725/189488397-0acb9e1c-7121-4fb9-8572-6afc3655f4c8.PNG)